src/app/SuccessButton.css

```diff
@@ -0,0 +1,3 @@
+.button {
+  background-color: green !important;

```

Comment:
I've updated PR with green color override. It still requires `!important`. Seems like selectors priority is still an issue here.